.github/workflows/pull_request.yml

```diff
@@ -12,9 +12,9 @@ jobs:
         with:
           python-version: 3.9
       - name: Install Dependencies
-        run: pip install -r requirements/local.txt
+        run: DOCKER_COMPOSE=docker-compose make install
```

Comment:
What's the reason for the `DOCKER_COMPOSE` command here? Does `docker compose` not exist in github CI?

Reply:
Yeah `docker compose` fails in github CI 😞

Reply:
Boooo lol. Maybe this can be set as an env var at the highest level? I think you can add an `env` tag under jobs and just set this there for all steps: https://docs.github.com/en/actions/reference/environment-variables

Reply:
Done in 8c9e4e5